README.rst

```diff
@@ -33,17 +33,25 @@ like bounding boxes.
 Installation
 ------------
 
-The ``geoxarray`` library will be available on PyPI and can be installed with
-pip::
+The ``geoxarray`` library isavailable on PyPI and can be installed with
```

Comment:
```suggestion
The ``geoxarray`` library is available on PyPI and can be installed with
```

Reply:
Wha...wha...a reviewer!?!? Thanks for catching this!

Reply:
Haha. I've been watching geoxarray for a long time. Excited to see where things are headed!